var/spack/repos/builtin/packages/heffte/package.py

```diff
@@ -26,10 +26,16 @@ class Heffte(CMakePackage, CudaPackage, ROCmPackage):
     version("2.3.0", sha256="27c0a8da8f7bc91c8715ecb640721ab7e0454e22f6e3f521fe5acc45c28d60a9")
     version("2.2.0", sha256="aff4f5111d3d05b269a1378bb201271c40b39e9c960c05c4ef247a31a039be58")
     version("2.1.0", sha256="527a3e21115231715a0342afdfaf6a8878d2dd0f02f03c92b53692340fd940b9")
-    version("2.0.0", sha256="12f2b49a1a36c416eac174cf0cc50e729d56d68a9f68886d8c34bd45a0be26b6")
-    version("1.0", sha256="0902479fb5b1bad01438ca0a72efd577a3529c3d8bad0028f3c18d3a4935ca74")
-    version("0.2", sha256="4e76ae60982b316c2e873b2e5735669b22620fefa1fc82f325cdb6989bec78d1")
-    version("0.1", sha256="d279a03298d2dc76574b1ae1031acb4ea964348cf359273d1afa4668b5bfe748")
```

Comment:
Should these versions also be deprecated instead of simply removed?  (see https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions)

Reply:
`0.1` and `0.2` are not really versions, they are pre-releases with very low quality in every aspect. It is OK to remove those, we are keeping the stable versions for now.

I've read the policy and those two absolutely should be removed.

Reply:
I returned the "version", but I hope really nobody is trying to use thoe.